src/main/java/org/osiam/resources/package-info.java

```diff
@@ -35,8 +35,6 @@
  *  <p>/Group - org.osiam.resources.controller.GroupController - is a Controller to create, replace, modify, get,
  *  delete and search groups.</p>
  *
- *  <p>/me - org.osiam.resources.controller.MeController - is a Facebook /me clone to get name, email and id of an user.</p>
```

Comment:
Because we are planning to get rid of the `package-info.java` files in the servers anyways. The is now inherent value in them.